.travis.yml

```diff
@@ -6,4 +6,8 @@ jdk:
 script:
   - git clone https://github.com/scala-exercises/site.git site
   - pushd site; sbt 'definitions/publishLocal' 'runtime/publishLocal' 'compiler/publishLocal' 'sbt-exercise/publishLocal' ; popd
-  - sbt test
\ No newline at end of file
+  - sbt test
+after_success:
+  - echo $KEYS_PASSPHRASE | gpg --passphrase-fd 0 keys.tar.gpg
+  - tar xfv keys.tar.gz
+  - sbt publishSigned
```

Comment:
Would this only happen for master builds? We don't want snapshots when people issue PRs and Travis kicks in.